pkg/ccl/changefeedccl/changefeed_test.go

```diff
@@ -10748,3 +10770,9 @@ func TestChangefeedProtectedTimestampUpdate(t *testing.T) {
 
 	cdcTest(t, testFn, feedTestForceSink("kafka"), withTxnRetries)
 }
+
+func assertReasonableMVCCTs(t *testing.T, ts string) {
```

Comment:
super nit: `Ts` should be capitalized like `TS` but given that would make it look like one long acronym, maybe `MVCCTimestamp`?